samples/snippets/fsharp/azure/blob-storage.fsx

```diff
@@ -5,31 +5,28 @@ open Azure.Storage.Blobs.Models
 open Azure.Storage.Blobs.Specialized
 open System.Text
 
+System.Environment.CurrentDirectory <- __SOURCE_DIRECTORY__
```

Comment:
It doesn't appear that we're referencing this explicitly;  we should consider a comment to explain that this is used implicitly when interacting with the `File` object.